src/lib/Server/Output/ValueObjectVisitor/CreatedPolicy.php

```diff
@@ -20,11 +20,9 @@ class CreatedPolicy extends Policy
     /**
      * Visit struct returned by controllers.
      *
-     * @param \Ibexa\Contracts\Rest\Output\Visitor $visitor
-     * @param \Ibexa\Contracts\Rest\Output\Generator $generator
      * @param \Ibexa\Rest\Server\Values\CreatedPolicy $data
      */
-    public function visit(Visitor $visitor, Generator $generator, $data): void
+    public function visit(Visitor $visitor, Generator $generator, mixed $data): void
```

Comment:
So after my suggestion for `src/lib/Server/Output/ValueObjectVisitor/Policy.php` this one pops up for PHPStan, but I think strict types help us here uncover composition bug. `CreatedPolicy` extends `ValueObject` and has inner `Policy` object as a composition. Therefore `CreatedPolicy extends Policy` is incorrect - `Policy` visitor should also be a composition.

See also that  in `src/bundle/Resources/config/value_object_visitors.yml:431` parent for `CreatePolicy` is set to `ValueObjectVisitor`, not `Policy` - which IMO is correct, the implementation needs not to extend Policy.